build/DownloadSonarPlugin/DownloadAndExtractSonarJS.cs

```diff
@@ -51,10 +51,10 @@ public class DownloadAndExtractSonarJS: Task
         //          package\package.json
 
         // The archive containing the eslintbridge
-        private const string SourceEsLintBridgeFilePattern = "eslint-bridge-*.tgz";
+        private const string SourceEsLintBridgeFilePattern = "sonarjs-*.tgz";
```

Comment:
This related to #3929.
At build time, we download the SonarJS jar, unzip it, and extract some files.
The name of one of the files we extract has changed, so this string needed to be updated.